src/api/payment-requests/payment-activities.md

```diff
@@ -49,11 +49,14 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 
 {% h4 Activity Types %}
 
-| Name    | Description                   |
-|---------|-------------------------------|
-| request | Payment Request was created.  |
-| payment | Payment Request was paid.     |
-| refund  | Payment Request was refunded. |
+| Name      | Description                                        |
+|-----------|----------------------------------------------------|
+| request   | [Payment Request][] was created.                   |
+| cancelled | [Payment Request][] was cancelled by the merchant. |
+| expired   | [Payment Request][] wasn't paid before time out.   |
+| declined  | [Payment Request][] was declined by the shopper.   |
+| payment   | [Payment Request][] was paid.                      |
+| refund    | Funds were returned to the shopper.                |
```

Comment:
Should this be: request, payment, refund, cancellation, expiry

Reply:
I think you've got 2 pieces of feedback here, one is the ordering and the other is the text in the type.

The ordering we can update no trouble.

When talking about activity, I expect language that implies actions that happened in the past. This is in contrast to your suggestion, cancellation and expiry describe a process that's happening, where as expired and cancelled describe activities that have happened.

Could you give me some more context? Come grab me.

Reply:
This has been updated.